content/en/serverless/installation/nodejs.md

```diff
@@ -255,7 +256,7 @@ Replace `<TAG>` with either a specific version number (for example, `{{< latest-
   - Set `DD_LAMBDA_HANDLER` to your original handler, for example, `myfunc.handler`.
```

Comment:
Do we want to add DATADOG_SITE here as well?

Reply:
I think this section deals with the command line parameters, not env vars.